tower-backend/src/main/resources/application-livedev.yml

```diff
@@ -50,8 +50,8 @@ grails:
 mail:
   from: "paolo.ditommaso+tower@gmail.com"
   smtp:
-    host: email-smtp.eu-west-1.amazonaws.com
-    port: 587
+    host: "${TOWER_SMTP_HOST:email-smtp.eu-west-1.amazonaws.com}"
```

Comment:
This was not meant to be here, it is not even documented in the commit. Nevertheless, it could be a nice improvement. My apologies.

Reply:
np